dev-lang/haxe/haxe-3.4.7.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="An open source toolkit based on a modern, strictly typed programming language"
+HOMEPAGE="https://haxe.org/"
+
+LICENSE="GPL-2+ MIT"
+SLOT="3"
+KEYWORDS="~amd64 ~x86"
+
+EGIT_REPO_URI="https://github.com/HaxeFoundation/${PN}.git"
+EGIT_COMMIT="${PV}"
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+# Can only be build withing a git tree, otherwise it bails out
```

Comment:
within?

Reply:
Yes. By "a git tree" i mean that the makefile will not work if it's not in the cloned repository (e.g. if you download the sources in an archive, it won't work).